Redesign: Decidim Design translations page (iconography)

greenwoodt opened the pull request:
#### :tophat: What? Why?
i18n translations for the Iconography page in Decidim Design page. 

#### :pushpin: Related Issues
- Related to #12460 

#### Testing
 1. Generate a new development_app
 2. Go to http://localhost:3000/design
 3.  Inspect the Iconography section
 4. See the page translated

### :camera: Screenshots

:hearts: Thank you!

alecslupu commented:
> Any ideas @alecslupu?

Yes, we will not add the icon description to the translated fields. At least not in this stage.

andreslucena reviewed (commented):
Just  skimming the  code, I think both of my comments in https://github.com/decidim/decidim/pull/12516#pullrequestreview-1900112187 apply here, can you check them out and  change them here please?

greenwoodt commented:
> Just skimming the code, I think both of my comments in [#12516 (review)](https://github.com/decidim/decidim/pull/12516#pullrequestreview-1900112187) apply here, can you check them out and change them here please?

Yes not a problem. However, according to @alecslupu, we're not going to open and merge this PR just yet correct?

andreslucena commented:
> we're not going to open and merge this PR just yet correct?

Let's merge this with the i18n changes that you're proposing, as that's an starting point. 

After we fully close https://github.com/decidim/decidim/issues/12460, then we'll define how to tackle the i18n changes for the icons and implement that.

greenwoodt commented:
> > we're not going to open and merge this PR just yet correct?
> 
> Let's merge this with the i18n changes that you're proposing, as that's an starting point.
> 
> After we fully close #12460, then we'll define how to tackle the i18n changes for the icons and implement that.

Refactored and fixed the broken link. Will convert from draft to open and await for a review.
